sandbox/OpenIddict.Sandbox.Avalonia.Client/readme.md

```diff
@@ -0,0 +1,482 @@
+# Avalonia sample app guide
```

Comment:
These samples are engineering samples - hence the name, "sandbox" - meant to be used to test OpenIddict features as they are being developed. They are not meant to be used by the general public, so this is likely not the right place for this guidance.